### ISSUE TITLE
Liveness of this project

### ISSUE THREAD

**Issue by javadba** [vegas-viz/Vegas#138]
Project looks great at first glance.. But digging deeper.. there are only two sets of commits in the past nine months and four issues closed in the past six..    Are the original/core committers no longer interested?

**Comment by oshikiri** [comment]
+1

@aishfenton @dbtsai

**Comment by thimmy687** [comment]
+1

**Comment by aishfenton** [comment]
Yes, not finding any time to work on this project unfortunately ;( 

Oshikiri, if you like I've can add you as a co-maintainer. I see you've been doing a lot of work on moving the project forward.

**Comment by virusdave** [comment]
Would be great, especially if a new maintainer could please publish the 2.12 jar apparently supported since February https://github.com/vegas-viz/Vegas/issues/106 (merged into master already)

**Comment by oshikiri** [comment]
@aishfenton I don't have the Sonatype credentials. Could I have it?

>You need to have ~/.sbt/0.13/sonatype.sbt configured with the Sonatype credientials (which core contributors should have)
https://github.com/vegas-viz/Vegas/blob/master/CONTRIB.md#releasing

**Comment by espears4sq** [comment]
@oshikiri Just wanted to note that I am happy to contribute to the project as well, big or small tasks, to help it gain momentum and develop more feature parity with matplotlib and more documentation / generic ability to handle different input data formats to plots (breeze vectors, dataframes, etc).

If you want to assign any "good first task" issues to me, I will get started!

**Comment by oshikiri** [comment]
Welcome, @espears4sq!

I think that adding specs to BasicPlots.scala (c.f. #65) is one of good first tasks.

**Comment by oshikiri** [comment]
@aishfenton @dbtsai Could you release a new version?

https://github.com/vegas-viz/Vegas/issues/138#issuecomment-407110699

**Comment by dbtsai** [comment]
@oshikiri I don't have the credential to cut a new release. @aishfenton can help out. Thanks.

**Comment by LorenzoCC** [comment]
@aishfenton This seems like a great library, any chance you could hand over the credentials to the contributors? Thanks

**Comment by javadba** [comment]
@aishfenton  seconded: please consider to take a few minutes to transfer the ownership.

**Comment by josemarfdc** [comment]
+1

**Comment by Quafadas** [comment]
+1

**Comment by trepidacious** [comment]
@oshikiri Might it be worth temporarily having a fork published under another name until publishing of this repository is resumed? This could even be linked from the readme I guess? I've not used this library yet but it looks great, I'm just not sure what the best way to get it is, I guess I'll fork and publish locally for now.
